engine/table/src/test/java/io/deephaven/engine/table/impl/QueryTableWhereTest.java

```diff
@@ -227,6 +228,7 @@ public void testWhereOneOfTwo() {
                 i(4, 6, 8).toTracking(), col("x", 21, 3, 4), col("y", 'x', 'c', 'f')));
     }
 
+    @Ignore
```

Comment:
Let's add a comment and refer to the existing issue https://github.com/deephaven/deephaven-core/issues/53